andela/haku-rc
# #150332431 fix application bugs

Fix application bugs

#### What does this PR do?
- Fix application bugs

#### Description of Task to be completed?
- Fixes the JobQueue error
- Fixes the fast-render package error
- Fix the product price not showing bug

#### How should this be manually tested?
- Start the app with the command `reaction` on your terminal.
- On page load visible product prices should now be seen

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- #150332431 Implement sorting and filtering of search results.

#### Screenshots (if appropriate)
<img width="1408" alt="screen shot 2017-08-17 at 4 38 46 pm" src="https://user-images.githubusercontent.com/12285426/29420702-a1baa86a-836a-11e7-83ea-79959e1706eb.png">

#### Questions:
- None
